js/carDescriptionChanger.js

```diff
@@ -0,0 +1,29 @@
+var CarLot = (function(carLot){
+//gets and activates the selected card
+	carLot.selectCard = function (event, color){
+		targetCard = event.currentTarget;
+		//set target card styles
+		targetCard.style.borderWidth = "5px";
+		targetCard.style.backgroundColor = `${color}`;
+		targetCard.style.color = "white";
+		targetCard.classList.add("selected");
```

Comment:
Other than the color, which comes from the data, the other attributes are better off being set in the CSS.
You can use the "selected" class to your advantage to set the different CSS when a card is clicked.

The background should be a different color form the border.